### PR TITLE
Bump devel testing version

### PR DESCRIPTION
**Pull request by jborean93**
##### SUMMARY
Bumps the version tested with devel and include an explicit 2.14 sanity/unit testing stage.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
ansible.windows

### CODE REVIEW

File: plugins/filter/quote.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+DOCUMENTATION:
+  name: quote
+  author:
+  - Jordan Borean (@jborean93)
+  short_description: Quotes argument(s) for various Windows shells
+  description:
+  - Quotes argument(s) for the various Windows command line shells.
+  - Defaults to escaping arguments based on the Win32 C argv parsing rules that M(ansible.windows.win_command) uses.
+  - Using C(shell='cmd') or C(shell='powershell') can be set to escape arguments for those respective shells.
```
Comment (briantist) [inline plugins/filter/quote.yml:12]:
```suggestion
  - Using I(shell='cmd') or I(shell='powershell') can be set to escape arguments for those respective shells.
```

File: plugins/filter/quote.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2022 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION:
+  name: quote
+  author:
+  - Jordan Borean (@jborean93)
+  short_description: Quotes argument(s) for various Windows shells
+  description:
```
Comment (briantist) [inline plugins/filter/quote.yml:9]:
```suggestion
    seealso:
    - module: ansible.windows.win_command
    - module: ansible.windows.win_shell
    description:
```

Comment (jborean93) [reply plugins/filter/quote.yml:9]: Is this indentation correct, seems like it needs to be back 2 spaces.

Comment (briantist) [reply plugins/filter/quote.yml:9]: I'd double check it in a real editor for sure... I thought the same thing in the preview, went to edit it, and then in edit I counted spaces and it seemed correct, so I didn't see how I could fix it.

I've seen the online suggestion thing screw this up before...

Comment (briantist) [reply plugins/filter/quote.yml:9]: actually, this was probably my screw-up, I just edited it back to 2 spaces

File: plugins/modules/async_status.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      description:
+      - If C(status), obtain the status.
+      - If C(cleanup), clean up the async job cache (by default in C(~/.ansible_async/)) for the specified job I(jid).
+      type: str
+      choices: [ cleanup, status ]
+      default: status
+  author:
+  - Ansible Core Team
+
+EXAMPLESL: |
```
Comment (briantist) [inline plugins/modules/async_status.yml:28]:
```suggestion
EXAMPLES: |
```

File: plugins/modules/async_status.yml [hunk 4]
```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2022 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# This is a stub to satisfy the GitHub doc action, see the builtin docs for better up to date info
+
+DOCUMENTATION:
+  module: async_status
+  version_added: historical
+  short_description: Obtain status of asynchronous task
+  description:
```
Comment (briantist) [inline plugins/modules/async_status.yml:10]:
```suggestion
  seealso: 
  - module: ansible.builtin.async_status
  description:
```

File: plugins/modules/async_status.yml [hunk 5]
```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2022 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# This is a stub to satisfy the GitHub doc action, see the builtin docs for better up to date info
```
Comment (briantist) [inline plugins/modules/async_status.yml:4]: I added a `seealso:` but maybe we can discuss with the docs folks about this page automatically render as or redirect to, the builtin.

@felixfontein any thoughts?

File: plugins/modules/setup.yml [hunk 6]
```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2022 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# This is a stub to satisfy the GitHub doc action, see the builtin docs for better up to date info
+
+DOCUMENTATION:
+  module: setup
+  version_added: historical
+  short_description: Gathers facts about remote hosts
+  options:
```
Comment (briantist) [inline plugins/modules/setup.yml:10]:
```suggestion
  seealso:
  - module: ansible.builtin.setup
  options:
```

File: plugins/modules/slurp.yml [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+      description:
+      - The file on the remote system to fetch. This I(must) be a file, not a directory.
+      type: path
+      required: true
+      aliases: [ path ]
+  notes:
+  - This module returns an 'in memory' base64 encoded version of the file, take
+    into account that this will require at least twice the RAM as the original file size.
+  seealso:
+  - module: ansible.builtin.fetch
```
Comment (briantist) [inline plugins/modules/slurp.yml:24]:
```suggestion
  - module: ansible.builtin.fetch
  - module: ansible.builtin.slurp
```

File: plugins/modules/async_status.yml (hunk 5 above)
Comment (jborean93) [reply plugins/modules/async_status.yml:4]: The seealso definitely helps. Having these entries should at least fix up these pages on the official docs

image

I'm fine with a redirect to keep things on the same page as it has been historically. Maybe having them separate might even be helpful when things start to differ from the Windows functionality and the POSIX variants.

**Review by briantist — APPROVED** [review]

**Comment by jborean93** [comment]
Thanks for the review @briantist and the fixes. I'll merge in the changes once CI is green but happy to continue the discussion around the redirects if need be. The more I think about it the more I feel that having separate docs will be helpful. It just wasn't possible before but now that it is we might as well take advantage of it.

**Comment by briantist** [comment]
> Thanks for the review @briantist and the fixes. I'll merge in the changes once CI is green but happy to continue the discussion around the redirects if need be. The more I think about it the more I feel that having separate docs will be helpful. It just wasn't possible before but now that it is we might as well take advantage of it.

I tend to agree! Was mostly suggesting redirect or something special based on the comment in the file. We should probably remove/change that comment as the last change if we want to ensure these docs are independent and kept up-to-date.

**Comment by jborean93** [comment]
Good point, I've removed the comment for now, can always revisit it later if need be.

**Review by briantist — APPROVED** [review]

**Pull request merged**